diff --git a/node_modules/@types/atom/index.d.ts b/node_modules/@types/atom/index.d.ts
index c2489cb..da9be3b 100644
--- a/node_modules/@types/atom/index.d.ts
+++ b/node_modules/@types/atom/index.d.ts
@@ -4214,6 +4214,8 @@ export interface Package {
 
     /** If a previous rebuild failed, get the contents of stderr. */
     getBuildFailureOutput(): string|null;
+
+    metadata: Record<string, any>;
 }
 
 /** Package manager for coordinating the lifecycle of Atom packages. */
@@ -4249,7 +4251,7 @@ export interface PackageManager {
 
     // General Package Data
     /** Resolve the given package name to a path on disk. */
-    resolvePackagePath(name: string): string|undefined;
+    resolvePackagePath(name: string): string|null;
 
     /** Is the package with the given name bundled with Atom? */
     isBundledPackage(name: string): boolean;
